packages/office-ui-fabric-react/src/components/ScreenReaderAlert/ScreenReaderAlert.Props.ts

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+
+export enum ReadingMode {
+  /**
+   * Do not read the content at all.
+   */
+  DoNotRead = 0,
+
+  /**
+   * The content in this component will read out after other reading tasks of screen reader.
+   * E.g. first read the information of currently focus control, then this content.
+   * NOTE: ReadingMode.ReadAfterOtherContent doesn't support ChromeVOX v53.
+   */
+  ReadAfterOtherContent = 1,
+
+  /**
+   * Read immediately after this component is triggered rendering and updated.
+   * Most screen readers will announce the word 'alert' before or after the whole sentence.
+   */
+  ReadImmediately = 2
+}
+
+export interface IScreenReaderAlertProps extends React.Props<HTMLElement> {
+  /**
+   * Use readingMode to specify whether to read the alert, or read before/after normal screen reader content.
+   *
+   * @default ReadingMode.ReadAfterOtherContent
+   */
+  readingMode?: ReadingMode;
+
+  /**
+   * Use indicator to specifically tell screen reader this text should be read even when text is not changed.
+   * Especially in case of re-reading the same text or the text that is possibly the same with the old one.
+   *
+   * Change the indicator to read the text no matter the text is changed or not.
+   * Keep indicator the same value or simply not use it to read ONLY when the text is changed.
+   */
+  indicator?: number;
```

Comment:
Thank you David, this comment is exactly what I tried to say. I will update the JSDoc.

- It also means the caller would need to store a numeric value in their state which is really awkward.
IMO, this is unavoidable. Imagine that user has a ScreenReaderAlert component inside their React component, and that component sometimes call its "setState" to update the UI but has NO change on ScreenReaderAlert string. In that case, how can we know the ScreenReaderAlert should be read out again or not?
Sometimes user just update the some other UI in their component and doesn't mean to trigger the ScreenReaderAlert to read again.
Sometimes user want to specifically read the same error message again.

The idea of exposing this props is to notify user that they can programmatically trigger the reading no matter the string is changed or not.